game/chat.go

```diff
@@ -312,12 +312,6 @@ func sendMsgNotificationsToFCM(ctx context.Context, host string, gameID *datasto
 		return err
 	}
 
-	dataPayload, err := NewFCMData(msgContext.fcmData)
-	if err != nil {
-		log.Errorf(ctx, "Unable to encode FCM data payload %v: %v; fix NewFCMData", msgContext.fcmData, err)
-		return err
-	}
-
```

Comment:
I've gotten rid of the data payload in notifications for now. I understand that they are used to power the click action on the client side.

I'm not convinced that it is the best approach, I'm going to re-implement the click actions later, and hopefully in a way that is a little more straight-forward. I was struggling a lot with the encoding and decoding of data. Felt like client knew too much about what was being sent to it. I reckon there's an easier way.